torch/utils/_sympy/functions.py

```diff
@@ -1200,7 +1200,9 @@ def eval(cls, a):
                 return getattr(sympy, name)(a)
             return None
 
-    OpaqueUnaryFn.__name__ = "OpaqueUnaryFn_" + name
+    nm = "OpaqueUnaryFn_" + name
+    OpaqueUnaryFn.__name__ = nm
+    OpaqueUnaryFn.__qualname__ = nm
```

Comment:
This works, but should qualname just be the same as name here?

Reply:
I actually tried it with the module qualifier and pickle had the audacity to break GRRR